pages/wasm/README.md

```diff
@@ -0,0 +1,43 @@
+![Netlify examples](https://user-images.githubusercontent.com/5865/159468750-df1c2783-39b2-40da-9c0f-971f72a7ea3f.png)
+
+# WebAssembly with Netlify Edge Functions
+
+Using WebAssembly, you can speed up key components of your server logic. Using Netlify Edge Functions, you can bring that to the edge. This allows developers to build more calculation intensive logic in C, C++, Rust, etc. and push it even closer to users. 
```

Comment:
I would prefer to leave out the etc... so it can read:

"This allows developers to build more calculation intensive logic in languages such as C, C++, and Rust"

And in both places this sentence appears.